scripts/snapshot/CVX/vote_proposal.py

```diff
@@ -0,0 +1,70 @@
+import requests
+import json
+import time
+
+from eth_account import messages
+from brownie import interface
+
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import registry
+
+SNAPSHOT_VOTE_RELAYER = "https://snapshot-relayer.herokuapp.com/api/message"
+
+SNAPSHOT_DEFAULT_HEADERS = {
+    "Accept": "application/json",
+    "Content-Type": "application/json",
+    "Referer": "https://snapshot.org/",
+}
+
+
+def main(proposal="QmSwrVFfQkgmvRiL2qUssk6XfGjoQs4tTRQPsrJyqCbtbR", choice=2):
+    test_msig = GreatApeSafe(registry.eth.badger_wallets.test_multisig_v1_3)
+
+    sign_message_lib = interface.ISignMessageLib(
+        registry.eth.gnosis.sign_message_lib, owner=test_msig.address
+    )
+
+    payload = {
+        "version": "0.1.3",
+        "timestamp": str(int(time.time())),
+        "space": "cvx.eth",
+        "type": "vote",
+        "payload": {
+            "proposal": proposal,
+            "choice": choice,  # json.dumps(choice_json, separators=(",", ":")),
+            "metadata": json.dumps({}),
```

Comment:
what does the `json.dumps` here do?

Reply:
it simply stringify the braces, but will be same if we leave it as **'{}'**. wanted to leave it like this as reminder that can accept later on for other occasion a dictionary with other info within it